faults/load/k6-loadgen/engine.yaml

```
@@ -0,0 +1,41 @@
+---
+apiVersion: litmuschaos.io/v1alpha1
+kind: ChaosEngine
+metadata:
+  name: nginx-chaos
+  namespace: default
+spec:
+  engineState: 'active'
+  chaosServiceAccount: litmus-admin
```

We need the app info for the target app as well:
```suggestion
  engineState: 'active'
  appinfo:
    appns: ''
    applabel: ''
    appkind: ''
  chaosServiceAccount: litmus-admin
```